app/_plugins/drops/plugins/hub_examples.rb

```diff
@@ -63,11 +32,7 @@ def render_kubernetes?
         end
 
         def render_terraform?
-          terraform_implemented? && @formats.include?(:terraform)
-        end
-
-        def terraform_implemented?
-          TERRAFORM_ENABLED_PLUGINS.include?(plugin_name)
+          @example.fetch('config', nil) && @formats.include?(:terraform)
```

Comment:
@fabianrbz This renders if there is a sample config provided (which is false for partners in most cases). _Really_ I'd like to add a condition that only renders if the maintainer is `kong-inc`. Any ideas?

Reply:
@mheap one quick  way would be to pass the `vendor` to the initialiizer.
It's available [here](https://github.com/Kong/docs.konghq.com/blob/main/app/_plugins/drops/plugins/hub_examples.rb#L118) in `schema.vendor` it returns a string that matches the folder name of the vendor, so you'll need to check if it matches `kong-inc` in the code